src/Rating/Rating.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import * as React from 'react';
+import Text from '../Text';
+import StyleSheet from '../PlatformStyleSheet';
+import type { StylePropType } from '../PlatformStyleSheet/StyleTypes';
+
+type Props = {|
+  +rating?: ?number,
+  +style?: StylePropType,
+|};
+
+export default function Rating({ rating, style }: Props) {
```

Comment:
I think I would prefer to have 
```suggestion
export default function Rating({ rating = 0, style }: Props) {
```
and not have `|| 0` below, for the logic to be a bit cleaner. And the type can now be
`+rating?: number`.
What do you think?

Reply:
Right, I'm also using default values in other components so it's definitely cleaner this way.